requirements.txt

```diff
@@ -1,6 +1,6 @@
 distlib==0.3.1
```

Comment:
We shouldn't need to track this any more since we moved to poetry - let's delete this file